packages/apidom-ns-workflows-1/src/elements/Reference.ts

```diff
@@ -0,0 +1,27 @@
+import { StringElement, ObjectElement, Attributes, Meta } from '@swagger-api/apidom-core';
+
+class Reference extends ObjectElement {
+  constructor(content?: Record<string, unknown>, meta?: Meta, attributes?: Attributes) {
+    super(content, meta, attributes);
+    this.element = 'reference';
+    this.classes.push('workflowsSpec-reference');
```

Comment:
Dash case convention for meta classes. Wouldn't `workflows-reference` suffice?
```suggestion
    this.classes.push('workflows-spec-reference');
```